connector_ebisumart/components/importer.py

```diff
@@ -319,5 +319,9 @@ class DelayedBatchImporter(AbstractComponent):
 
     def _import_record(self, external_id, job_options=None, **kwargs):
         """ Delay the import of the records"""
-        delayable = self.model.with_delay(**job_options or {})
+        job_options = job_options or {}
+        job_options['description'] = (
+            f'JOB: Import an Ebisumart record ({self.model._description})'
```

Comment:
I think this may help users identify which record has failed to be processed. What do you think?
```suggestion
            f"JOB: Import an Ebisumart record ({self.model._description}: {external_id})"
```
By the way, I don't like how single and double quotations are mixed in this module very much. I prefer to stick to the practice of using double quotations.